### PR TITLE
Feature: APP-2710 - Implement Avatar component

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description

- implement the Avatar component with stories and tests

Task: [APP-2710](https://aragonassociation.atlassian.net/browse/APP-2710)

## Type of change

-   [x] New feature (non-breaking change which adds functionality)

## Checklist:

-   [x] I have selected the correct base branch.
-   [x] I have performed a self-review of my own code.
-   [x] I have commented my code, particularly in hard-to-understand areas.
-   [x] I have made corresponding changes to the documentation.
-   [x] My changes generate no new warnings.
-   [x] Any dependent changes have been merged and published in downstream modules.
-   [x] I ran all tests with success and extended them if necessary.
-   [x] I have updated the `CHANGELOG.md` file in the root folder of the package after the [UPCOMING] title and before
        the latest version.
-   [x] I have tested my code on the test network.

[APP-2710]: https://aragonassociation.atlassian.net/browse/APP-2710?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by RuggeroCino — COMMENTED** [review]
👏🏻

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 -   Minimum `tailwindcss` version required
 
+### Added
+
+-   Implement `Avatar` component
```
Comment (RuggeroCino) [inline CHANGELOG.md:16]: We should already have an `added` section on the `main` branch with the `InputDate` component

**Review by RuggeroCino — APPROVED** [review]
Thanks @Fabricevladimir, I only left one additional comment for the `changelog` file ✅

File: CHANGELOG.md (hunk 1 above)
Comment (Fabricevladimir) [reply CHANGELOG.md:16]: Huh, I wonder why it's not conflicting 🤔

**Pull request merged**